lib/git/keybindings.sh

```diff
@@ -27,13 +27,13 @@ if [[ "$git_keyboard_shortcuts_enabled" = "true" ]]; then
       # Uses emacs style keybindings, so vi mode is not supported for now
       if ! set -o | grep -q '^vi .*on$'; then
         if [[ $shell == "zsh" ]]; then
-          _bind "$git_commit_all_keys"              " git_commit_all""\n"
-          _bind "$git_add_and_commit_keys"          " \033[1~ git_add_and_commit ""\n"
-          _bind "$git_commit_all_with_ci_skip_keys" " \033[1~ APPEND='[ci skip]' git_commit_all ""\n"
+          _bind "$git_commit_all_keys"              " git_commit_all""^M"
+          _bind "$git_add_and_commit_keys"          " \033[1~ git_add_and_commit ""^M"
+          _bind "$git_commit_all_with_ci_skip_keys" " \033[1~ APPEND='[ci skip]' git_commit_all ""^M"
         else
-          _bind "$git_commit_all_keys"              "\" git_commit_all\n\""
-          _bind "$git_add_and_commit_keys"          "\"\C-A git_add_and_commit \n\""
-          _bind "$git_commit_all_with_ci_skip_keys" "\"\C-A APPEND='[ci skip]' git_commit_all \n\""
+          _bind "$git_commit_all_keys"              "\" git_commit_all^M\""
+          _bind "$git_add_and_commit_keys"          "\"\C-A git_add_and_commit ^M\""
+          _bind "$git_commit_all_with_ci_skip_keys" "\"\C-A APPEND='[ci skip]' git_commit_all %M\""
```

Comment:
Is this supposed to be %M?

Reply:
`^M` works correctly for me.

Reply:
Ack, sorry. On my screen that line was not in view when I was reading the comment that wasn't on my screen. That is indeed a typo. Fix pushed.